### PR TITLE
Display reviews/ratings on details pages and format files

### PR DESCRIPTION
**Pull request by raysarinas**

### CODE REVIEW

File: client/src/components/lists/CourseReviewList.tsx [hunk 1]
```diff
@@ -1,28 +1,43 @@
-import React from "react"
-import { Card, CardColumns, Col, Container, ListGroup, ListGroupItem, Row, Tab, Tabs, } from "react-bootstrap";
+import React from "react";
```
Comment (raysarinas) [inline client/src/components/lists/CourseReviewList.tsx:1]: no changes in this file, just formatted

File: client/src/components/lists/InstructorReviewList.tsx [hunk 2]
```diff
@@ -1,29 +1,46 @@
-import React from "react"
-import { Card, CardColumns, Col, Container, ListGroup, ListGroupItem, Row, Tab, Tabs, } from "react-bootstrap";
+import React from "react";
```
Comment (raysarinas) [inline client/src/components/lists/InstructorReviewList.tsx:1]: no changes in this file, just formatted

File: client/src/components/lists/RequisiteList.tsx [hunk 3]
```diff
@@ -1,40 +1,47 @@
...
+        </ListGroupItem>
+        <ListGroupItem>
+          <Card>
+            <Card.Header as="h5">Postrequisites</Card.Header>
+            {props.course.postreqs.length > 0 ? (
+              props.course.postreqs.map((postreq: string) => {
+                return <Card.Body>{postreq}</Card.Body>;
+              })
+            ) : (
+              <Card.Body></Card.Body>
```
Comment (raysarinas) [inline client/src/components/lists/RequisiteList.tsx:40]: this is the only change (other stuff is formatting) I made - just mapping the list of strings onto the page so it isn't cluttered

File: client/src/pages/InstructorPage.tsx [hunk 4]
```diff
@@ -1,57 +1,82 @@
...
+              ? reviews.map((review: InstructorReview) => (
+                  <InstructorReviewDetails review={review} />
+                ))
+              : showNoReviewsCard()}
+          </Col>
+        </Row>
+      </Container>
+    </div>
+  );
+}
```
Comment (raysarinas) [inline client/src/pages/InstructorPage.tsx:82]: these are the only changes in this file - the rest is just formatting

**Review by aelgohar — APPROVED** [review]

**Pull request merged**